diktat-rules/src/test/kotlin/org/cqfn/diktat/util/FixTestBase.kt

```diff
@@ -113,4 +202,9 @@ open class FixTestBase(protected val resourceFilePath: String,
 
         return testComparatorUnit.compareFilesFromFileSystem(expected, actual)
     }
+
+    companion object {
+        private const val KTLINT_VERSION = "0.46.1"
```

Comment:
Ideally, these versions should be set in `pom.xml` and passed into test as properties. Maybe not in this PR, but we should not forget to implement it in the future

Reply:
Ok. I will create a issue for this when this pr is merge